documentation/modules/rn-27-resolved-issues.adoc

```diff
@@ -7,6 +7,17 @@
 
 {project-first} 2.7 has the following resolved issues:
 
+[id="resolved-issues-2-7-3_{context}"]
+== Resolved issues 2.7.3
+
+.Migration plan does not fail when conversion pod fails
+
+In earlier releases of {project-short}, when running the `virt-v2v` guest conversion, the migration plan did not fail if the conversion pod failed, as expected. This issue has been resolved in {project-short} 2.7.3. link:https://issues.redhat.com/browse/MTV-1569[(MTV-1569)]
+
+.Large number of VMs in the inventory can cause the inventory controller to panic
+
+In earlier releases of {project-short}, having a large number of virtual machines (VMs) in the inventory could cause the inventory controller to panic and return a `concurrent write to websocket connection` warning. This issue was caused by the concurrent write to the WebSocket connection and has been addressed by the addition of a lock, so the Go `routine` *waits* before sending the response from the server. link:https://issues.redhat.com/browse/MTV-1220[(MTV-1220)]
+
```

Comment:
```suggestion

.Migration fails with module `network-legacy` configured in RHEL guests

In earlier releases of {project-short}, migration is failed if the module configuration file was available in the guest and `dhcp-client package` was not installed, returning a `dracut module 'network-legacy' will not be installed, because command 'dhclient' could not be found` error. This issue has been resolved in {project-short} 2.7.3. link:https://issues.redhat.com/browse/MTV-1615[(MTV-1615)]

```